src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(n -> Arrays.stream(n.split(",")))
+                .map(Integer::parseInt)
+                .filter(n -> n % 2 == 0)
```

Comment:
Please avoid using 1-2 letter variable naming even in streams. After couple of weeks you won't remember what is "n", "s", "sb" and so on.
```suggestion
                .flatMap(stringValues -> Arrays.stream(stringValues.split(",")))
                .map(Integer::parseInt)
                .filter(number -> number % 2 == 0)
```
Please recheck whole solution for this issue